bgpd/bgp_flowspec_util.c

```diff
@@ -579,5 +578,21 @@ int bgp_flowspec_match_rules_fill(uint8_t *nlri_content, int len,
 				 __func__, type);
 		}
 	}
+	if (bpem->match_packet_length_num || bpem->match_fragment_num ||
```

Comment:
I worry a little about the long term maintenance and understanding of this huge if/or statement... I can't think of any clear way to do this that might be cleaner, except perhaps a switch statement with fall throughs for each BGP_PBR type... @donaldsharp -- what do you think?

Reply:
contributing to the discussion: 
- There will be policy decisions in BGP, since the same flowspec rule may be implemented either by 'ip rule ' or by 'netfilter'. So, the border may move.
- next enhacnements to come:  add a 'ip rule with same table, with 2 NHs different ( kind of ECMP based pbr).